line/line.mjs

```diff
@@ -1,27 +1,86 @@
-import * as utils from '../utilities/shared.mjs'
+import * as utils from '../utilities/shared.mjs';
 
-export async function findLineById(id = null) {
-  let line = null
+export async function findLineById(id = null, options = {}) {
+  let line = null;
+
+  if (id === null || id === undefined) {
+    return line;
+  }
 
   if (!utils.validateID(id)) {
-    return line
+    return line;
   }
 
   const mockPause = new Promise((resolve) => {
     setTimeout(() => {
-      resolve(null)
-    }, 1500)
-  })
+      resolve(null);
+    }, 1500);
+  });
 
   const linesArray = [
-    { id: 123, text: "Hey TPEN Works on 123" },
-  ]
+    { id: 123, text: 'Hey TPEN Works on 123', '@context': 'http://t-pen.org/3/context.json', '@type': 'Annotation', creator: 'https://store.rerum.io/v1/id/hash', project: '#ProjectId', canvas: 'https://example.com/canvas.json', layer: '#AnnotationCollectionId', viewer: 'https://static.t-pen.org/#ProjectId/#PageId/#LineId123', license: 'CC-BY' }
+  ];
 
-  line = linesArray.find((line) => line.id === id) || null
+  line = linesArray.find((line) => line.id === id) || null;
 
   if (line === null) {
-    line = await mockPause
+    line = await mockPause;
+  }
+
+  if (line !== null) {
```

Comment:
you can unwrap this `if` your guard clause saved you a whole indent!

Reply:
updated in the code